maxim/logger/livekit/audio_input.py

```
@@ -0,0 +1,49 @@
+            f"[{self.__class__.__name__}] {hook_name} failed; error={str(e)}\n{traceback.format_exc()}"
+        )
+
+
+def post_hook(self, result, hook_name, args, kwargs):
+    try:
+        scribe().debug(
+            f"[{self.__class__.__name__}] {hook_name} completed; result={result}"
+        )
+    except Exception as e:
+        scribe().error(
+            f"[{self.__class__.__name__}] {hook_name} failed; error={str(e)}\n{traceback.format_exc()}"
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)